### PR TITLE
Fix: [#13334] Uninitialised variables in CustomTabDesc

### PR DESCRIPTION
**Pull request by Sadret**

### CODE REVIEW

File: distribution/changelog.txt [hunk 1]
```diff
@@ -43,7 +43,7 @@
 - Fix: [#13266] Plugin API: Deleting key of sharedStorage not working.
 - Fix: [#13278] Desync caused by ghost tiles changing the ride mode.
 - Fix: [#13289] Litter and vomit sometimes not loading with RCT1 saved game or scenario
-- Fix: [#13292] Impossible excitement rating requirements with finish building 5 coasters goal
+- Fix: [#13334] Uninitialised variables in CustomTabDesc
```
Comment (Gymnasiast) [inline distribution/changelog.txt:46]: Looks like you made a mistake here. But a release is imminent, so it's better to wait with fixing and rebasing.

**Comment by Sadret** [comment]
Why can't it auto merge the changelog?

**Comment by Gymnasiast** [comment]
Because it needs a few lines on either side that are the same. You don't want it to automerge where it shouldn't.

**Comment by Sadret** [comment]
Ok, good to know.

**Pull request merged**